tests/unittests/graphOptzTest.cpp

```diff
@@ -872,3 +872,49 @@ TEST_F(GraphOptz, concatElim) {
   // Check that the concat node is gone.
   EXPECT_EQ(countNodeKind(F_, Kinded::Kind::ConcatNodeKind), 0);
 }
+
+// Check the transformation Concat(Reshape(x) * N) -> Reshape(Concat(x * N)).
+TEST_F(GraphOptz, concatReshapes) {
+  const size_t shape[] = {10, 20};
+  const size_t reshape[] = {10, 1, 20};
+  Type t(ElemKind::FloatTy, shape);
+  std::vector<NodeValue> inputs;
+  for (size_t i = 0; i < 50; i++) {
```

Comment:
50 is impressive, but do we need that many for a test? I guess 10 should be enough ;-)

Reply:
changed to 10